core/dcschema_enum_instances.mcf

```diff
@@ -2185,3 +2185,11 @@ typeOf: dcs:FBI_MeasurementQualifierEnum
 Node: dcid:SummerSeason
 name: "SummerSeason"
 typeOf: dcs:ClimaticSeasonEnum
+
+Node: dcid:LocalCurrency
+name: "LocalCurrency"
```

Comment:
Might be useful to have descriptions for these?

Reply:
+1

Reply:
this could also go into a new `currency.mcf` file.